dotcom-rendering/src/components/Crosswords.editions.tsx

```diff
@@ -1,14 +1,11 @@
-import Crossword from '@guardian/react-crossword';
+import { Crossword } from '@guardian/react-crossword';
+import type { CAPICrossword } from '@guardian/react-crossword/dist/@types/CAPI';
```

Comment:
Non-blocking: we should probably avoid importing from this directory. In the article crossword player we have something like this to get access to this type:

```tsx
import type { CrosswordProps } from '@guardian/react-crossword';

type CAPICrossword = CrosswordProps['data'];
```